node/src/components/chain_synchronizer/error.rs

```diff
@@ -126,4 +130,27 @@ pub(crate) enum Error {
         #[serde(skip_serializing)]
         JoinError,
     ),
+
+    /// Error fetching a trie.
+    #[error(transparent)]
+    FetchTrie(
+        #[from]
+        #[serde(skip_serializing)]
+        FetchTrieError,
+    ),
+}
+
+#[derive(Error, Debug)]
+pub(crate) enum FetchTrieError {
+    /// Fetcher error.
+    #[error(transparent)]
+    FetcherError(#[from] FetcherError<TrieOrChunk>),
+
+    /// Trie was being fetched from peers by chunks but was somehow fetch from storage.
+    #[error("Trie was being fetched from peers by chunks but was somehow fetched from storage. Perhaps there are parallel downloads going on?")]
```

Comment:
Does this work?
```suggestion
    #[error("Trie was being fetched from peers by chunks but was somehow fetched from storage. \
             Perhaps there are parallel downloads going on?")]
```

Reply:
Fixed in [f07d887](https://github.com/casper-network/casper-node/pull/2697/commits/f07d88729f65861a2c09df9beff7f72f5ab8147f)